src/main/java/com/sprint/mission/discodeit/service/repository/ChannelRepositoryInterface.java

```diff
@@ -0,0 +1,30 @@
+package com.sprint.mission.discodeit.service.repository;
+
+import com.sprint.mission.discodeit.entity.Channel;
+
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+
+public interface ChannelRepositoryInterface {
```

Comment:
Interface는 제거해도 의미가 전달될 것 같아요.

전 보통 메서드명에 abstract나 Interface 포함시키지 않는 편인데요,  
객체가 가지는 책임에 집중하면 그 객체가 추상클래스인지 인터페이스인지 중요하지 않다고 생각하기 때문이에요.

```suggestion
public interface ChannelRepository {
```